packages/darklang/stdlib/option.dark

```diff
@@ -248,6 +248,4 @@ module Darklang =
         (options: List<PACKAGE.Darklang.Stdlib.Option.Option<'a>>)
         : List<'a> =
         PACKAGE.Darklang.Stdlib.List.filterMap options (fun option ->
-          match option with
-          | Some v -> PACKAGE.Darklang.Stdlib.Option.Option.Some v
-          | None -> PACKAGE.Darklang.Stdlib.Option.Option.None)
\ No newline at end of file
+          Fun.identity option)
```

Comment:
Ah, this isn't quite what I meant. I mean that we could do this:

```
List.filterMap options (fun option -> option)
```

which should be the same as this:

```
List.filterMap options Fun.identity
```

Reply:
Oh yeah, sorry. Fixed